lib/cassandra_migrations/railtie/tasks.rake

```diff
@@ -65,10 +67,10 @@ namespace :cassandra do
   end
 
   namespace :test do
-    desc 'Load the development schema in to the test keyspace'
+    desc 'Load the development schema in to the test keyspace via a full reset'
     task :prepare do
       Rails.env = 'test'
-      Rake::Task['cassandra:reset'].execute
+      Rake::Task['cassandra:migrate:reset'].execute
```

Comment:
We could also leave around the `setup` task with a deprecation notice whenever it runs, and have that call the new `cassandra:migrate:reset` task. Gives any dependent users time to change before it is removed.

Say the word and I'll add this code.